src/client/client4.ts

```diff
@@ -3300,11 +3300,13 @@ export default class Client4 {
     // This function belongs to the Apps Framework feature.
     // Apps Framework feature is experimental, and this function is susceptible
     // to breaking changes without pushing the major version of this package.
-    executeAppCall = async (call: AppCall) => {
-        call.context.user_agent = 'webapp';
+    executeAppCall = async (call: AppCallRequest, type: AppCallType) => {
+        const callCopy = JSON.parse(JSON.stringify(call)) as AppCallRequest;
+        callCopy.path = `${callCopy.path}/${type}`;
```

Comment:
should we check for `! callCopy` ?

Reply:
I do not foresee any case where callCopy may return undefined. If call is indeed an AppCallRequest, and it is not undefined, callCopy should be always defined. Any other case I may be missing?

Reply:
Not really, just defensive coding, every time we do an `as X` we should ask the q imo.